stdlib/grpc/src/main/java/org/ballerinalang/net/grpc/builder/components/AbstractStub.java

```diff
@@ -85,8 +82,12 @@ public void setDescriptors(Set<Descriptor> descriptors) {
         return descriptors;
     }
 
-    public void setMessageList(List<Message> messageList) {
-        this.messageList = messageList;
+    public void setMessageMap(List<Message> messageList) {
```

Comment:
Shall we keep the method name as `setMessageList` here? 
Since we are setting a message list, I feel the previous name more suits.

Reply:
Kept it as setMessageMap, but passed a map instead of a list as the parameter